packages/core/src/app/customer/Customer.tsx

```diff
@@ -171,28 +172,30 @@ class Customer extends Component<CustomerProps & WithCheckoutCustomerProps & Ana
             isExecutingPaymentMethodCheckout = false,
             isInitializing = false,
             isSubscribed,
+            isWalletButtonsOnTop,
             privacyPolicyUrl,
             requiresMarketingConsent,
             providerWithCustomCheckout,
             onUnhandledError = noop,
             step,
             useFloatingLabel,
         } = this.props;
+        const checkoutButtons = isWalletButtonsOnTop
+          ? null
+          : <CheckoutButtonList
+            checkEmbeddedSupport={checkEmbeddedSupport}
+            deinitialize={deinitializeCustomer}
+            initialize={initializeCustomer}
+            isInitializing={isInitializing}
+            methodIds={checkoutButtonIds}
+            onError={onUnhandledError}
+          />
 
         return (
             providerWithCustomCheckout === PaymentMethodId.StripeUPE ?
                 <StripeGuestForm
                     canSubscribe={canSubscribe}
-                    checkoutButtons={
-                        <CheckoutButtonList
-                            checkEmbeddedSupport={checkEmbeddedSupport}
-                            deinitialize={deinitializeCustomer}
-                            initialize={initializeCustomer}
-                            isInitializing={isInitializing}
-                            methodIds={checkoutButtonIds}
-                            onError={onUnhandledError}
-                        />
-                    }
+                    checkoutButtons={checkoutButtons}
```

Comment:
@AlanMunozN do we need to make some adjustments for wallet buttons on top of checkout approach. Since with the current approach of experiment and setting is on then wallet buttons will be visible even if shopper authenticates with stripe link.